xknxproject/loader/project_loader.py

```diff
@@ -172,12 +173,37 @@ def _create_device(
 
         return device
 
+    @staticmethod
+    def __get_links_from_ets4(com_object: ElementTree.Element) -> str | None:
+        # Check if "Connectors" is available. This will always fail for ETS5/6
+        if (connectors := com_object.find("{*}Connectors")) is None:
+            return None
+
+        # Send GA is the primary GA, put it first in the list
+        if (send_ga := connectors.find("{*}Send")) is None:
+            return None
+
+        # Remove the project ID from GA
+        links = [send_ga.get("GroupAddressRefId", "").split("_")[1]]
+        # Receive GA are additional GA, add them at the end of the list
+        for receive_ga in com_object.findall("{*}Receive"):
+            links.append(receive_ga.get("GroupAddressRefId", "").split("_")[1])
+
+        # Return a list of GA as string as for "Links" in ETS5/6
+        return " ".join(links)
+
     @staticmethod
     def _create_com_object_instance(
         com_object: ElementTree.Element,
     ) -> ComObjectInstanceRef | None:
         """Create ComObjectInstanceRef."""
-        if not (links := com_object.get("Links")):
+
+        # Check if "Links" is available for ETS5/6
+        if (links := com_object.get("Links")) is None:
+            # ETS4 has a different schema to define links, check if it is available
+            links = _TopologyLoader.__get_links_from_ets4(com_object)
```

Comment:
Could be nicer to have a `TopologyLoader` and a `TopologyLoaderETS4` that overrides `_create_com_object_instance` but since everything is static here, I didn't feel like refactoring everything. The right instance could be selected in `ProjectLoader` then based on the schema version.

Reply:
We can store Schema version in `KNXProjContents`. That is passed anyway and we can just select the appropriate method by that with the `is_ets_4_project` helper.

Reply:
Do you mean pass `KNXProjContents` to `_TopologyLoader.load()` (and further down)? Indeed, this could be a nicer solution :+1:

Reply:
right, there is not really a need to only have `@staticmethod`. We could just create an instance 
like
```py
ProjectLoader(
            knx_proj_contents=self.knx_proj_contents,
            space_usage_names=space_usage_names,
        ).load()
```
instead of
```py
ProjectLoader.load(
            knx_proj_contents=self.knx_proj_contents,
            space_usage_names=space_usage_names,
        )
```